packages/esm-service-queues-app/src/patient-search/visit-form/visit-form.component.tsx

```diff
@@ -314,24 +309,22 @@ const VisitForm: React.FC<VisitFormProps> = ({ patientUuid, toggleSearchType, cl
               </ContentSwitcher>
             )}
             {config.showRecommendedVisitTypeTab && contentSwitcherIndex === 0 && !isLoading && (
```

Comment:
Presumably we can get rid of the isLoading check here, as this is handled by the skeleton within the selector component?